cmd/core/main.go

```diff
@@ -151,3 +163,45 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// setupChecks is used to create readiness&liveness probes
+func setupChecks(mgr ctrl.Manager) {
+	setupLog.Info("creating readiness/health check")
+	if err := mgr.AddReadyzCheck("ping", healthz.Ping); err != nil {
+		setupLog.Error(err, "unable to create ready check")
+		os.Exit(1)
+	}
+
+	if err := mgr.AddHealthzCheck("ping", healthz.Ping); err != nil {
+		setupLog.Error(err, "unable to create health check")
+		os.Exit(1)
+	}
+}
+
+// waitSecretVolume waits for webhook secret ready to avoid mgr running crash
+func waitSecretVolume(certDir string) {
```

Comment:
Can the pod start when the volume doesn't exist?

Reply:
ths pod will be stuck in "containercreating" unless the volume is mounted successfully.